### PR TITLE
Remove frozen state and unify readonly tasks

### PR DESCRIPTION
**Pull request by thescouser89**
Previously the ReadOnly task had a separate view from the general tasks
view. This commit unifies readonly with the tasks view so that you can
mark a task as 'readonly' from the task view. Fancy!

The frozen state was removed. Instead, a task is 'frozen' when it is not
'cronjob active'. I know. Confusing. Oh well...

### CODE REVIEW

**Comment by thescouser89** [comment]
No merge yet till I figure out how much it'll break stuff :)

File: app/models/package.rb [hunk 1]
```diff
@@ -19,7 +23,10 @@ class Package < ActiveRecord::Base
       500 => {:status => 'QUEUED FOR TEST', :style => 'background-color: #b2f4ff;'},
       -1 => {:status => 'DUPLICATE', :style => 'background-color: #b2ffa1;'}
   }
-  acts_as_textiled :notes
+  if Rails::VERSION::STRING < "4"
```
Comment (liweinan) [inline app/models/package.rb]: COOL

**Comment by liweinan** [comment]
Looks great migration method Dustin :-) Please feel free to merge it when you feel it's proper. Thanks for contribution to Rails4 upgrade!

**Comment by thescouser89** [comment]
Woops just realized my form_for hack never actually worked on Rails2 :( Gonna make some commits to get it working now.

**Comment by thescouser89** [comment]
Closed since there are some bugs introduced with migrations. will fix them one by one.

**Pull request closed without merging**